config.js

```diff
@@ -248,213 +229,164 @@ exports.updateFromArgv = function() {
                 + `please visit https://github.com/online-go/gtp2ogs/tree/devel`
                 + `\nDebug status: ${debugStatus}`);
     // B - check deprecated argv //
-    testDeprecatedArgv(optimist.argv, "komis");
+    testDeprecatedArgv(argv, "komis");
```

Comment:
What is the use for passing a hardcoded "komis" parameter? Can't we inline this in the function?